### ISSUE TITLE
Cathedral Concept - WACCF Patch

### ISSUE THREAD

**Issue by juliocabanaboy** [loot/skyrimse#1182]
Hi Gang,
First a quick, **YOU GUYS ARE AWESOME! THANKS**
![Cathedral Concept - WACCF Patch](https://user-images.githubusercontent.com/67127665/85079846-c87b7900-b195-11ea-8343-4348713ac722.jpg)
I have been searching online for an answer and found this:
**_"Update: That is exactly it. The Cathedral - Armory / WACCF patch LOOT recommended is not only outdated, but completely unnecessary. That's what's calling "Cathedral Concept - Weapons Armor and Clothing.esp". Thanks for pointing that out, I would've ran in circles for days otherwise."_**
It can be found here:
https://www.reddit.com/r/skyrimmods/comments/cgacc6/esp_missing_after_adding_patches_cathedral_armory/eufqi51/

I hope this helps.
Cheers

**Comment by MacSplody** [comment]
There's no message for `WACCF_CathedralConceptWAC_Patch.esp` in the current LOOT masterlist.
The message you're seeing above is from MO2, you'll receive those if you have a plugin with a missing master.

**Comment by EvilDylan** [comment]
Cathedral Concept - Weapons Armor and Clothing (Now Cathedral - Armory) _used to_ have an ESP because it used to include Unique Uniques--this is the only reason it needed an ESP. Unique Uniques was removed from the mod in version 2.0. Thus, the WACCF patch is trying to patch for Unique Unique's model changes, but since those model changes are no longer part of Cathedral Armory it doesn't have a plugin to patch.

Basically the WACCF patch is redundant now and you shouldn't have it.

**Comment by juliocabanaboy** [comment]
You are awesome! Thanks so much for your help.

Sent from Yahoo Mail on Android 
 
  On Sat., 20 Jun. 2020 at 6:25 pm, EvilDylan<notifications@github.com> wrote:   


Cathedral Concept - Weapons Armor and Clothing (Now Cathedral - Armory) used to have an ESP because it used to include Unique Uniques--this is the only reason it needed an ESP. Unique Uniques was removed from the mod in version 2.0. Thus, the WACCF patch is trying to patch for Unique Unique's model changes, but since those model changes are no longer part of Cathedral Armory it doesn't have a plugin to patch.

Basically the WACCF patch is redundant now and you shouldn't have it.
